domains/juhan.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Juhan280",
+           "email": "",
+           "discord": "748758747861745796",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.bl7nmvrp2Y5bQdpY7efBQgF_x7nWwmt3hQ0CuaT_zDsK9OHc5roDOM6p2xAeVb8XIX7dWBYx2d7x-y3--EWeUBnZF9bZz6S7R2tCX67o2bJ2pr0Cx_-sZQNVT6lrf88-v-pNrValI6m5n8SemrCjp-MkLRaYY65byY1e1rccyLK_JzMzK_CHlLv8KKVqm0Z3TM5hSjvoufHO61RTZtkXveLqnFnn5PJjDyUXW5xf1RWbo7WcDhCcGdrzA9KSAaG42IL4sg8jsRa5q1Pxsk9er0PLsAS2QNW5fmJTTMn95bgVhLvu_spW4rO3XSAQV5fojuJuB9pYuv5TFfMeDgqKCQ.lGmjA-PFLydXyFRyCp6JOA.qfHOSFz15kYiu-BRJuRou9eP9ywX1vphAApO-tEDhJvdxE-KRNW74w2tVcgukusUbtSEUUottnvmqMoi39IDYyarb6EhBu2iROuwHcrEBkM.Xwvko9cO1zjrwsNlKUW8mQ"
+        },
+    
+        "record": {
+            "CNAME": "juhan280.github.io"
```

Comment:
Little content present on the site.

Reply:
Ok, I will open the PR once i add more content